File: Python/ExerciciosHerancPolimorfismo/Seção16/ex02/Person.py
class Person:
    def __init__(self, name, age, heigth):
        self.__name = name
        self.__age = age
        self.__heigth = heigth

    @property
    def name(self):
        return self.__name

    @name.setter
    def name(self, value):
        self.__name = value

    @property
    def age(self):
        return self.__age

    @age.setter
    def age(self, value):
        self.__age = value

    @property
    def heigth(self):
        return self.__heigth

    @heigth.setter
    def heigth(self, value):
        self.__heigth = value

File: Java/CursoJava/secao16/ex02/person.java
package ex02;

public class person {
    private String name;
    private String age;
    private String heigth;

    public person(String name, String age, String heigth){
        this.name = name;
        this.age = age;
        this.heigth = heigth;
    }

    public String getName(){
        return name;
    }

    public String getAge(){
        return age;
    }

    public String getHeigth(){
        return heigth;
    }

    public void setName(String name){
        this.name = name;
    }

    public void setAge(String age){
        this.age = age;
    }

    public void setHeigth(String heigth){
        this.heigth = heigth;
    }
}
